chore(report): REST APIs week 1  & 2 report

valens200 opened the pull request:
## Project

REST API improvements.

### Changes
- Changed the folder structure by adding the folder holding my updates for REST API improvements.
- Added updates for weeks 1, 2, and 3 for REST API improvements.

GMishx commented on docs/2024/text-phrases/index.md:
```diff
@@ -3,29 +3,51 @@ sidebar_position: 1
 
-- [Email](mailto:email.here)
-- [LinkedIn](https://linkedin.com/in/my-user)
+- [Email](valensniyonsenga2003@gmail.com)
```
Please change to email link as it causes build failure.

```suggestion
- [Email](mailto:valensniyonsenga2003@gmail.com)
```

valens200 replied:
Sure @GMishx I will take a look into this.

GMishx reviewed (changes requested):
Minor change needed.

@valens200 please rebase the branch with main and move other files created by @dvjsharma as well.

valens200 commented:
> Minor change needed.
> 
> @valens200 please rebase the branch with main and move other files created by @dvjsharma as well.
Thank you @GMishx, I have rebased the branch and moved other files from Sharma.
